src/components/article/articleCard/index.js

```diff
@@ -27,6 +27,10 @@ export class ArticleCard extends Component {
                 <i className="fa fa-comments-o" /> 
                 {" "}
                 {24}
+                &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
```

Comment:
Instead of the repeated `&nbsp;`, is there a property you can apply here?

Reply:
Okay let me find out what I can apply and get back to you

Reply:
I have fixed this, thanks.